README.md

```diff
@@ -21,14 +21,14 @@ You are free to add more content and style it anyway you want. Below is some [ex
         1. `<title>`
         1. `<link>` including a favicon
     1. `<h1>` with your name
-    1. `<section>` with a brief description of yourself with the class name `about-me`
-    1. `<img>` containing an image of yourself or your avatar with the class name `profile-img`
-    1. `<ul>` containing all of your current and upcoming projects
-    1. `<footer>` containing a list of links to all your public social media accounts (including GitHub). If you don't want to share your own social media accounts feel free to use the Black Codher accounts.
-1. External stylesheet
+    1. `<section>` containing a brief description of yourself, with the class name `about-me`
+    1. `<img>` containing an image of yourself or your avatar, with the class name `profile-img`
+    1. `<ul>` containing all of your current and upcoming projects, with the class name `projects`
+    2. `<footer>` containing a `<ul>` with the class name `social-media`, listing all your public social media accounts (including GitHub). If you don't want to share your own social media accounts feel free to use the Black Codher accounts.
```

Comment:
I'll need to check what we have in the slides. Because `id`s have a higher specificity it can lead to problems when adding rules to  elements with lower specificity, so could form good habits in the students early to not rely on using `id`...but at the same time uniqueness 🤷🏾‍♀️ .

I'll check the slides. Hopefully that'll help make a decision

Reply:
OK, cool. I was debating whether to set the class-based tests as “at least one” rather than “exactly one”, but that didn’t seem to fit the criteria as given. I’ll wait for your decision.

Reply:
@tanyapowell did you make a call on this?

Reply:
nope, just use `id`s